packages/clay-icon/src/index.tsx

```diff
@@ -28,7 +28,7 @@ const ClayIcon: React.FunctionComponent<IProps> = ({
 	spritemap,
 	symbol,
 	...otherProps
-}) => {
+}: IProps) => {
```

Comment:
I dont think this should be necessary since we already declare `IProps` in line 26.

Reply:
hey @bryceosterhaus this is more because of react-docgen to be able to generate the API's in doc, it can not infer the props from `React.FunctionComponents <IProps>` for example. Probably this could be an improvement there...

Reply:
Yes, considering Matu comment I gonna stay with this type.

Reply:
dang, thats unfortunate about react-docgen. I wonder if we could send a feature for this or if its on the roadmap. Otherwise, we should decide how we want to declare these types since this area will sort of be a "gotcha"

Reply:
Yeah, I think we can try to send a resource there, I'm not sure if we can implement our own resolver in that case, if so, we can solve it from here but I think it should be a react-docgen resource. I'll try to check this out and come back here again.

I did not want to waste too much time on it, so it was the quickest solution I had found, but it sure should not be anything permanent like that.